config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultImagePath            = "/usr/share/clear-containers/clear-containers.img"
 	defaultHypervisorPath       = "/usr/bin/qemu-lite-system-x86_64"
 	defaultProxyURL             = "unix:///run/cc-oci-runtime/proxy.sock"
+	defaultPauseBinPath         = "/tmp/bundles/pause_bundle/rootfs/bin/pause"
```

Comment:
Yes I agree it would be better somewhere else. But the goal of this PR was to get everything working. What are you suggesting as a default location ?
And as a follow up PR, we could have this path as a parameter in the toml config file.